src/main/java/crawlercommons/sitemaps/sax/DelegatorHandler.java

```diff
@@ -107,19 +112,48 @@ private void startRootElement(String uri, String localName, String qName, Attrib
         // the <rss> or <rdf> tag is a <channel> tag, so we can use that.
         // See https://github.com/crawler-commons/crawler-commons/issues/87
         // and also RSS 1.0 specification http://web.resource.org/rss/1.0/spec
-        else if ("channel".equals(localName)) {
+        else if ("rss".equals(localName)) {
+            return; // ignore
```

Comment:
Is this related to improving namespace support?

Reply:
Ok, I remember: at an early stage I've added this to fix that also rss feeds are checked for namespace and then are always rejected because of the now removed else-if-condition three lines below:
```
Failed to parse sitemap http://www.globes.co.il/webservice/rss/rssfeeder.asmx/FeederNode?iID=2173: crawlercommons.sitemaps.UnknownFormatException: Namespace  does not match standard namespace http://www.sitemaps.org/schemas/sitemap/0.9
```
That's fixed anyway with this PR. Either I forgot about it or kept it to avoid the debug log message below: "Skipped unknown root element <rss> in ..."

I'll remove it and open a new issue to review skipping of "unknown" elements until the "root" element is found:
- it should also be done for <rdf>
- while for <rss> and <rdf> we need to step down into the tree and wait for <channel>, this shouldn't be done for other "unknown" tags. E.g., from the debug log when an XHTML document is parsed as sitemap (it's valid XML):
```
18/06/05 11:10:32 DEBUG sitemaps.SiteMapParser:124 - Skipped unknown root element <html> in ...
18/06/05 11:10:32 DEBUG sitemaps.SiteMapParser:124 - Skipped unknown root element <head> in ...
18/06/05 11:10:32 DEBUG sitemaps.SiteMapParser:124 - Skipped unknown root element <title> in ...
18/06/05 11:10:32 DEBUG sitemaps.SiteMapParser:124 - Skipped unknown root element <meta> in ...
```